City-of-Helsinki/drupal-helfi-platform-config
# UHF-4929 block library

# block library [UHF-4929](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4929)
Paragraph list view changes and single paragraph library item permissions

## What was done
* Paragraph list view configurations updated: 
1. title link removed
2. block language column was removed from the table,  
* Paragraph library item canonical url redirected to entity usage page.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-4929_block_library`
* Run `make drush-updb drush-cr`

## How to test
* Paragraph view changes:
1. Go to /admin/content/paragraphs

What you should see:
- Title should not be a link but a text
- There shouldn't be a language column in the table

Select one of the existing blocks
1. Go to edit page
2. Click "näytä/view" (which would normally take you to the paragraphs page)

What should happen after you click "näytä/view":
- You should be redirected to entity usage page.

## Other PRs
* (Already reviewed / deployed) https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/121
